src/routers/mailingList/validator.ts

```diff
@@ -4,9 +4,8 @@ const validator = (route: string) => {
   switch (route) {
     case '/':
       return [
-        check('email').normalizeEmail().isEmail(),
+        check('email').isString(),
         check('interestedEvents').isArray(),
-        check('interestedEvents.*').notEmpty(),
```

Comment:
Why'd we lose this boy? 😢

Reply:
Oh that was my bad. Why do we have that boy? Can't the array be empty?

Reply:
Ohhhhhhhhhhh this checking that the elements in the array weren't empty? Okay my b we can have em